### DISCUSSION TITLE
Add xadapter support

### DISCUSSION THREAD

**Discussion by umisetokikaze** [lllyasviel/stable-diffusion-webui-forge#292] [Ideas]
The codebase for xadapter was recently made public. Although I'm not an engineer and cannot grasp the details, it seems possible to maintain compatibility between XL and 1.5, such as with Lora. Additionally, it appears to support controlnet right from the samples. Hence, I'd like to request the implementation of native support or an extension for xadapter in sd-forge.

https://github.com/showlab/X-Adapter/tree/main?tab=readme-ov-file

**Comment by huchenlei** [comment]
Already tracked by #290
